package.json

```diff
@@ -5,8 +5,8 @@
         "build": "d2-app-scripts build",
         "start": "d2-app-scripts start",
         "start:nobrowser": "BROWSER=none d2-app-scripts start",
-        "test": "d2-app-scripts test --coverage",
-        "test:watch": "d2-app-scripts test --watch",
+        "test": "TZ=Etc/UTC d2-app-scripts test --coverage",
+        "test:watch": "TZ=Etc/UTC d2-app-scripts test --watch",
```

Comment:
I would just mock this in the test instead of globally. Otherwise it's not clear why this is being mocked.

Reply:
See how we do it here for example: https://github.com/dhis2/ui/blob/master/components/select/src/multi-select/__tests__/selection-list.test.js#L10-L29

Reply:
The time zone is being mocked as part of the mock of the `useTimeZoneConversion` hook, so we don't actually need to set it generally for the test (I think this is left over from some other approach I did not use). I've just removed this.